src/olympia/abuse/cinder.py

```diff
@@ -290,6 +291,11 @@ def get_attributes(self):
             'name': self.get_str(self.addon.name),
             'slug': self.addon.slug,
             'summary': self.get_str(self.addon.summary),
+            'promoted': self.get_str(
+                promoted_group.name if promoted_group != NOT_PROMOTED else ''
```

Comment:
`NOT_PROMOTED` is falsey so we can do
```suggestion
                promoted_group.name if promoted_group else ''
```
(and drop the import above)

Reply:
https://github.com/mozilla/addons-server/pull/22095/commits/c9270b04f989f34ff7fe68149e6e7c5aaaea5fc7